tools/taskcluster/upload_symbols.sh

```diff
@@ -1,11 +1,6 @@
 #/bin/sh
-git clone git@github.com:MozillaReality/symbolgenerator.git tools/taskcluster/symbols && cd tools/taskcluster/symbols && git rebase origin/master && cd ../../..
+#git clone git@github.com:MozillaReality/symbolgenerator.git tools/taskcluster/symbols && cd tools/taskcluster/symbols && git rebase origin/master && cd ../../..
 
```

Comment:
It looks like you will not clone the script when doing a new run? Why? and How you get the script at a new image?

Reply:
Good catch, I comment it out for testing.